packages/flutter_tools/lib/src/bundle.dart

```diff
@@ -19,7 +19,7 @@ const String defaultMainPath = 'lib/main.dart';
 const String defaultAssetBasePath = '.';
 const String defaultManifestPath = 'pubspec.yaml';
 String get defaultDepfilePath => fs.path.join(getBuildDirectory(), 'snapshot_blob.bin.d');
-String get defaultApplicationKernelPath => fs.path.join(getBuildDirectory(), 'app.dill');
```

Comment:
whether this the dill path with extension should be
app.dill or app.track.dill depends on trackWidgetCreation.